src/auth/oauth2.js

```diff
@@ -26,7 +26,7 @@ const createAuthProvider = ( name, baseUrl, userUrl, redirectUrl, clientId ) =>
 		if ( localStorage.getItem( REQUEST_TOKEN_STORAGE_KEY ) ) {
 			localStorage.removeItem( REQUEST_TOKEN_STORAGE_KEY );
 			localStorage.setItem( TOKEN_STORAGE_KEY, token );
-			window.history.replaceState( {}, null, window.location.pathname );
+			window.location = window.location.pathname;
```

Comment:
I refresh the page after loginSuccess, the user object (avatar etc..) was not showing up on login success, unless I refresh.